.eslintrc

```diff
@@ -22,6 +22,12 @@
     "btoa": true
   },
   "rules": {
+    "class-methods-use-this": [
```

Comment:
* [ ] How come we're using `eslint-plugin-react` and still need a rule like this? We should check the config.